docs/resources/mcs_db_cluster.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: "mcs"
+page_title: "mcs: db_cluster"
+sidebar_current: "docs-db-cluster"
+subcategory: ""
+description: |-
+  Manages a db cluster.
+---
+
+# mcs\_db\_cluster (Resource)
+
+Provides a db cluster resource. This can be used to create, modify and delete db cluster for galera_mysql and postgresql datastores.
+
+## Example Usage
+
+```terraform
+
+resource "mcs_db_cluster" "mydb-cluster" {
+  name        = "mydb-cluster"
+
+  datastore {
+    type    = "postgresql"
+    version = "12"
+  }
+
+  cluster_size = 3
+
+  flavor_id   = example_flavor_id
+
+  volume_size = 10
+  volume_type = example_volume_type
+
+  network {
+    uuid = example_network_id
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the cluster. Changing this creates a new cluster
+
+* `datastore` - (Required) Object that represents datastore of the cluster. Changing this creates a new cluster. It has following attributes:
+    * `type` - (Required) Type of the datastore. Changing this creates a new cluster. Type of the datastore can either be "galera_mysql" or "postgresql".
+    * `version` - (Required) Version of the datastore. Changing this creates a new cluster.
+
+* `cluster_size` - (Required) The number of instances in the cluster.
+
+* `keypair` - Name of the keypair to be attached to cluster. Changing this creates a new cluster.
+
+* `floating_ip_enabled` - Boolean field that indicates whether public access to the cluster is enabled. Changing this creates a new cluster.

```

Comment:
Boolean field that indicates whether floating ip is created for cluster or instance.

Reply:
Fixed